panda_rover/lidar_msg[] box_estimates